app/models/docker_container_wizard_states/exposed_port.rb

```diff
@@ -0,0 +1,16 @@
+module DockerContainerWizardStates
+  class ExposedPort < Parameter
+    ExposedPort.reset_callbacks(:validate)
+
+    belongs_to :environment,  :foreign_key => :reference_id, :inverse_of => :exposed_ports,
+                              :class_name => 'DockerContainerWizardStates::Environment'
+    validates :name,  :uniqueness => { :scope => :reference_id },
+                      :numericality => { :only_integer => true,
+                                         :greater_than => 0,
+                                         :less_than_or_equal_to => 165_35,
+                                         :message => "%{value} is not a valid port number" }
```

Comment:
`:message` here and below are not really necessary, Rails will provide a suitable message, and these ones will require translation.